tests/e2e/local/setup_localregistry.sh

```diff
@@ -0,0 +1,50 @@
+# !/bin/bash
+
+# File path to local registry for e2e tests
+LOCALREG_FILE=${ISTIO}/istio/tests/e2e/local/localregistry/localregistry.yaml
+
+echo "Deploying local registry onto cluster..."
+kubectl apply -f ${LOCALREG_FILE}
+
+count=0
+echo "Waiting for registry proxy to be active..."
+until $(kubectl -n kube-system get pods | grep -q '^kube-registry-proxy.*1/1.*$'); do
+    printf '.'
+    sleep 1
+    let count+=1
+    if [[ "$count" -gt 10 ]]; then
+        echo "Time limit reached. Fail to bring local registry proxy up."
+        exit 1
+    fi
+done
+echo "Local registry proxy is up"
+
+count=0
+echo "Waiting for registry to be active..."
+until $(kubectl -n kube-system get pods | grep -q '^kube-registry-v0.*1/1.*$'); do
+    printf '.'
+    sleep 1
+    let count+=1
+    if [[ "$count" -gt 10 ]]; then
+    	echo "Time limit reached. Fail to bring local registry up."
+    	exit 1
+    fi
+done
+echo "Local registry is up"
+
+# Set up docker daemon for linux
+case "$OSTYPE" in
+  darwin*)  
```

Comment:
Add an instruction for user to enable it in docker daemon?

Reply:
It's documented in UsingGKE.md/UsingBluemix.md and README.md's in local setup already.